cumulus_library/builders/psmpy_lite.py

```diff
@@ -0,0 +1,593 @@
+# The following code is adapted from https://github.com/adriennekline/psmpy,
+# less the graphing elements and with some light production-y adaptations.
+# Most of the commenting is from the existing library, the get_* functions
+# are returning datasets that would be processed for charts in psmpy.
```

Comment:
I might add a chunk here noting that this is licensed under MIT and give the copyright holder's name (Adrienne Kline) - I know you link to the repo, but it's nice to have it local too - who knows when that repo will shutter for some reason.

Reply:
Yeah good call on the license.

I'm not sure about storing the repo in the project - we could keep a fork in our org?